src/main/java/com/coherentsolutions/training/aqa/java/zhavrid/util/IEDownload.java

```diff
@@ -0,0 +1,40 @@
+package com.coherentsolutions.training.aqa.java.zhavrid.util;
+
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.ie.InternetExplorerOptions;
+
+import java.io.IOException;
+
+public class IEDownload {
```

Comment:
see the comments for ff and chrome. Let's use your knowledge base from java core